docs/src/pages/demos/lists/CheckboxList.js

```diff
@@ -44,6 +44,7 @@ class CheckboxList extends React.Component {
           {[0, 1, 2, 3].map(value => (
             <ListItem
               key={value}
+              role={undefined}
```

Comment:
Could ListItem take care of this?

Reply:
No, it's specific to having the Checkbox inside.

Reply:
👍

Reply:
I believe it's not important and that 98% of the people won't care. But why not. I couldn't see an easy way to document it.